todo_app/templates/index.html

```diff
@@ -10,9 +10,32 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
+      <form action="/complete" method="POST">
+        {% for item in items %}
+        <div class="form-check">
+          {% if item['status'] == 'Completed' %}
+          <input class="form-check-input" type="checkbox" value="{{ item.id }}" name="completed" id="{{ item.id }}" checked disabled>
+          {% else %}
+          <input class="form-check-input" type="checkbox" value="{{ item.id }}" name="completed" id="{{ item.id }}"> 
+          {% endif %}
+          <label class="form-check-label" for="{{ item.id }}">{{ item.title }}</label>
+        </div>
+        {% endfor %}
+        <button type="submit" class="btn btn-primary">Complete</button>
+      </form>
+    </div>
+  </div>
+  <br/>
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Add New Item</h2>
+      <form action="/" method="POST">
+        <div class="form-group">
+          <label for="item_title">Item title:</label>
+          <input type="text" id="item_title" name="item_title"><br>
```

Comment:
Minor, but adding e.g. a `required` tag to this input element would help our browser know to not let users accidentally submit an empty title (though without server-side validation it's just user assistance, rather than absolute prevention)